activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Correctly handle frozen options when serializing ActiveRecord objects.
+
+    * Brian Abreu *
```

Comment:
@kamipo Sorry for the typo in the changelog. I cleaned that up, rebased, and pushed the corrected changes.